docs/redirects.csv

```diff
@@ -567,7 +567,7 @@ https://help.expensify.com/articles/expensify-classic/settings/account-settings/
 https://help.expensify.com/articles/expensify-classic/connections/Additional-Travel-Integrations,https://help.expensify.com/articles/expensify-classic/connections/Travel-receipt-integrations
 https://help.expensify.com/articles/new-expensify/getting-started/Upgrade-to-a-Collect-Plan,https://help.expensify.com/Hidden/Upgrade-to-a-Collect-Plan
 https://help.expensify.com/articles/expensify-classic/bank-accounts-and-payments/payments/Reimburse-Reports-Invoices-and-Bills,https://help.expensify.com/articles/expensify-classic/bank-accounts-and-payments/payments/Reimburse-Reports
-https://help.expensify.com/articles/new-expensify/expenses-&-payments/pay-an-invoice.html,https://help.expensify.com/articles/new-expensify/expenses-&-payments/Pay-an-invoice
+https://help.expensify.com/articles/new-expensify/expenses-and-payments/pay-an-invoice.html,https://help.expensify.com/articles/new-expensify/expenses-and-payments/Pay-an-invoice
```

Comment:
The `sourceURL` got updated on some of these 😅 which means the redirect won't work

Reply:
massive face palm 😅